spec/services/spotify_service_spec.rb

```diff
@@ -19,7 +19,7 @@
   end
 
   describe ".get_recent_albums_for" do
-    let(:user) { User.new(access_token: "BQCglneaf-osQ50FKcfsV0HRbABrfdF2PhZ9diu5LfFQC9canNvukI-xcMYcaGp5rjsC4NaMblc21d7CvI5YohxLLfKBPeLVwGDTtT5693-PEcyuMQXZfYX6shPy3c-bXgOy4y9m7VG--jZgNCopIhkX09x4Rqb9ECjdXqo4FPn52oaqPAlYB8WnkrBmHwdgQ6iV5amiz8x-MMaLMuo36-HsdOQI95vS4WO2CfCnQ3gf-FKTQrlX1SXnwNeFuvnV1DJYeEPVAJgFBcPZOrS712zuQmWJ-Pk") }
+    let(:user) { User.new(access_token: "BQBQPOPfqMewO4GN65W8PYV_fB-CLecFbkd_9xNdkm7m1ScR24gU8TMccz2Ld0ePQD1HX-umnvrfk7IYPNF-HrGpeaosC6eQlai9OSF5XFmtzqt9Rb1ni6jhZ7XsAfTTMkHV3xvK4tiYkPtB07nWxGvdo7aRllhYZ8LtDV3Whnwo-3YzvFhXz1fj3JhBvJeOgBURmS0HMT8GvLuaJH1tWnGsXYwfwt24fyHvmA6YTJ0-lCfFQrGNl3_jITDJNhetYkd9nBNl_970iqV7h2ZDdbqFGrcGr5cGqQ") }
```

Comment:
Lets kill the token.

Reply:
Yep! Nuked it and a few other things as well.